crates/sui-types/src/effects/effects_v2.rs

```diff
@@ -472,12 +484,10 @@ impl TransactionEffectsV2 {
         }
     }
 
-    pub fn new_with_tx_and_status(tx: &SenderSignedData, status: ExecutionStatus) -> Self {
-        Self {
-            transaction_digest: tx.digest(),
-            status,
-            ..Default::default()
-        }
+    pub fn new_with_tx_and_status_for_test(tx: &SenderSignedData, status: ExecutionStatus) -> Self {
```

Comment:
Can this function be removed (also new_with_tx_for_test)? Looks like there is no more call site.